src/sparta/checkers/SimpleFlowAnnotatedTypeFactory.java

```diff
@@ -328,6 +328,22 @@ public AnnotationMirror getTopAnnotation(AnnotationMirror start) {
         }
 
         @Override
+        public AnnotationMirror getBottomAnnotation(AnnotationMirror start) {
+            if (start.toString().contains("Sink")) {
```

Comment:
I'm not sure why the original code looks like this. I'll try to use instanceof in both methods, and see whether we passed the test.